### DISCUSSION TITLE
Authentication

### DISCUSSION THREAD

**Discussion by sergi0g** [outline/outline#5098] [Self Hosting]
Hi, I've just discovered outline and it looks like just the tool I need. I want to self-host it through the docker container, but after reading the documentation, I've come to the conclusion that I can't sign in with just an email and a password and I have to use oauth. I don't plan to expose my server to the internet, so I would like to ask if there is way to:
- Enable email/password authentication (not link-based)
- Setup my own authentication server (Do I need https? How do I set it up)

If this is possible, or you have something else to suggest, please provide *detailed* instructions. I've tried reading other discussions, but I cannot find a clear answer. Please help me

**Comment by glasgowm148** [comment]
Doesn't seem like it. 

Very odd that email/password is not the default tbh.

**Reply by sergi0g** [reply]
I agree, but I'm asking for a solution here. There should be a way for it to be enabled.
